src/main/java/org/bytedeco/javacv/Java2DFrameConverter.java

```diff
@@ -409,33 +409,33 @@ public static void flipCopyWithGamma(DoubleBuffer srcBuf, int srcStep,
     }
 
     public static void applyGamma(Frame frame, double gamma) {
-        applyGamma(frame.image[0].position(0), frame.imageDepth, frame.imageStride, gamma);
+        applyGamma(frame.image[0], frame.imageDepth, frame.imageStride, gamma);
     }
     public static void applyGamma(Buffer buffer, int depth, int stride, double gamma) {
         if (gamma == 1.0) {
             return;
         }
         switch (depth) {
             case Frame.DEPTH_UBYTE:
-                flipCopyWithGamma(((ByteBuffer)buffer).asReadOnlyBuffer(), stride, (ByteBuffer)buffer, stride, false, gamma, false, 0);
+                flipCopyWithGamma(((ByteBuffer)buffer).asReadOnlyBuffer(), 0, stride, (ByteBuffer)buffer, 0, stride, false, gamma, false, 0);
```

Comment:
The starting indices are not always going to be 0, let's leave them at position() like they were.

Reply:
Or maybe not, since we've been ignoring it, it might be better to keep it that way.

Reply:
Actually the method name is not explicit for this use case. Actually I read it as "applyGamma on Buffer with depth, stride and gamma", not "applyGamma on Buffer at this position with depth, stride....."

In any way is not safe to use position as we figured